Library/Formula/clang-format.rb

```diff
@@ -0,0 +1,46 @@
+class ClangFormat < Formula
+  homepage "http://clang.llvm.org/docs/ClangFormat.html"
+  version "2014-12-03"
+
+  stable do
+    # The 2014-12-03 tag is not imported in the github mirror, but resolves to this SHA.
+    url "https://github.com/llvm-mirror/llvm/archive/67351a195b7f7980b5a40f06222743585c5fd1f5.tar.gz"
```

Comment:
I see. Bottling is indeed probably the best fix as the compile still is a bit lengthy. Switched to svn+https URLs.